<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CopyPlane</class>
 <widget class="QDialog" name="CopyPlane">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>196</width>
    <height>96</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>196</width>
    <height>96</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QCheckBox" name="copyTileCol">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>110</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Copy Tile Collision</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="copyChunkPlane">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>34</y>
     <width>136</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Copy Chunks Tile Plane</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="confirmButton">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>59</y>
     <width>166</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>confirmButton</sender>
   <signal>accepted()</signal>
   <receiver>CopyPlane</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>confirmButton</sender>
   <signal>rejected()</signal>
   <receiver>CopyPlane</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
